scripts/globals/spells/dread_spikes.lua

```diff
@@ -16,12 +16,18 @@ function onSpellCast(caster,target,spell)
     local duration = 180;
     local power = 0;
     local typeEffect = EFFECT_DREAD_SPIKES;
+    local drainAmount = target:getHP() / 2;
```

Comment:
Whoops, fixed. I guess I spaced out when I read half of current max HP, since max HP usually doesn't change... though DRK does have Drain 2.